thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/completeness/checker/Wo4WAvgDataCompletenessAlgorithm.java

```diff
@@ -41,6 +41,7 @@
 /**
  * This is the implementation of the WO4W Average function or checking data completeness of datasets
  */
+@Deprecated
 public class Wo4WAvgDataCompletenessAlgorithm implements DataCompletenessAlgorithm {
 
   public static double DEFAULT_EXPECTED_COMPLETENESS = 80;
```

Comment:
This class could be deleted.
Let's delete everything under /completeness.checker

Reply:
The issue is that  `Wo4WAvgDataCompletenessAlgorithm` is used as default value for `DatasetConfigBean.dataCompletenessAlgorithm`. I understand that it is not used, but i am not sure about the impact if i remove this field from `DatasetConfigBean` and still have this field inside the JSON in DB.